### DISCUSSION TITLE
3d reconstruction from two images captured with Ricoh Theta V camera

### DISCUSSION THREAD

**Discussion by zengletian1491** [stella-cv/stella_vslam#114] [Q&A] [answered]
openvslam can be used to implement 3d reconstruction with videos. My questions are :
1)Can I use openvslam to implement dense 3d reconstruction from two images captured with Ricoh Theta V camera?
2)Can I use openvslam to implement sparse 3d reconstruction from two images captured with Ricoh Theta V camera?
3)Are there any materials or papers about this topic? I could find them in the referenced paper 《OpenVSLAM: A Versatile Visual SLAM Framework》.
Thank you in advance.

**Comment by mirellameelo** [comment, marked as answer]
@zengletian1491 Just to make sure, do you mean dense (or semi-dense) 3D reconstruction?

**Reply by zengletian1491** [reply]
I mean dense 3d reconstruction. I'm new to openvslam and I will denote the questions more clearly.Thank you.

**Reply by mirellameelo** [reply]
There are some people who used this Ricoh Theta V camera with OpenVSLAM and succeeded. Since you'd be running in a monocular configuration, you'd need at least two frames to start the mapping, but is important to mention that the mapping can require more frames to be able to initialize. Having only two is a "risk", but you can try it... Honestly, if the 3D reconstruction is your focus, I'd recommend you to use a direct visual SLAM, such as LSD-SLAM (I don't know if it runs with the camera you have though). Feature-based SLAMs (Openvslam case) are so more focused on localization... Hopefully,  you can find some improvements on dense reconstruction with post-ORBSLAM works.
